sdks/go/test/integration/primitives/pardo.go

```diff
@@ -18,10 +18,11 @@ package primitives
 import (
 	"flag"
 	"fmt"
-
 	"github.com/apache/beam/sdks/v2/go/pkg/beam"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/register"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/testing/passert"
+	"sync/atomic"
+	"time"
```

Comment:
Please put the standard library imports in a separate group at the top of the imports. For some reason the break was dropped, and the grouping changed.